egs/ami/s5/local/nnet3/run_lstm.sh

```diff
@@ -14,9 +14,29 @@ mic=ihm
 use_sat_alignments=true
 affix=
 speed_perturb=true
-splice_indexes="-2,-1,0,1,2 0"
 common_egs_dir=
 
+# lstm options
+splice_indexes="-2,-1,0,1,2"
+num_lstm_layers=1
+cell_dim=1024
+hidden_dim=1024
+recurrent_projection_dim=256
+non_recurrent_projection_dim=256
+truncated_sequence_width=20
+context_sensitive_chunk_width=20
```

Comment:
Wouldn't it be more consistent with the code to call this example_left_context or left_context?